projects/packages/jetpack-mu-wpcom/src/features/wpcom-wpadmin-page-view/wpcom-wpadmin-page-view.php

```diff
@@ -0,0 +1,188 @@
+<?php
+/**
+ * WPCom Tracking for WP Admin Page Views.
+ *
+ * @package automattic/jetpack-mu-wpcom
+ */
+
+namespace Automattic\Jetpack\Jetpack_Mu_Wpcom\Wpcom_Wpadmin_Page_View;
+
+use WPCOMSH_Support_Session_Detect;
+
+/**
+ * Bump a Tracks stat for every wp-admin page view not generated by an Automattician.
+ * This is possibly interesting on its own but will also be used as part of an
+ * initiative to understand how much supply we're offering for live chat demand.
+ * See https://nosaraproject.wordpress.com/2015/03/06/live-chat-stats/
+ */
+function wpcom_nosara_track_admin_page_views() {
+	if ( do_not_track_a11ns() ) {
+		return;
+	}
+
+	$is_simple_site = defined( 'IS_WPCOM' ) && IS_WPCOM;
+	$is_atomic_site = ! $is_simple_site;
```

Comment:
This is not necessarily true in Jetpack. It would be safer to use:
`$is_atomic_site = defined( 'IS_ATOMIC' ) && IS_ATOMIC`

Reply:
It shouldn't load on Jetpack sites. The code lives in jetpack-mu-wpcom package which is bundled only on WPCOM and Atomic sites.